src/kirby/scss/web/_ionic.scss

```diff
@@ -2,38 +2,59 @@
 @import "../base/functions";
 
 @function _get-kirby-color($variant) {
-    $map: $kirby-colors;
-    @if map-has-key($map, 'kirby-' + $variant) {
-        @return map-get($map, 'kirby-' + $variant);
-    }
+  $map: $kirby-colors;
+  @if map-has-key($map, "kirby-" + $variant) {
```

Comment:
Any reason for changing single `'` to `"`?

Reply:
I don't think I modified this file myself, must have come from a merge at some point. Furthermore, I can see we use " all over the place in that file